### ISSUE TITLE
README.rst latex rendering

### ISSUE THREAD

**Issue by enzbus** [enzbus/project_euromir#1]
Seems like Github lacks support for rendering LaTex, which is fully supported by rst since 2011 :(

https://github.com/github/markup/issues/83

**Comment by mwt** [comment]
Note, while GitHub rst does not support proper math blocks, it does support the nonstandard markdown syntax for math mode using $ and $$ (issue https://github.com/github/markup/issues/274). Maybe that helps you?

**Comment by enzbus** [comment]
Thanks, I tend to recycle ``.rst`` code for both the README and the sphinx documentation, so using Github specific syntax could be problematic. I'll try, thank you!

**Comment by mwt** [comment]
Makes sense. I do the same thing. I think there's a plugin for sphinx that allows for this notation, but I haven't used it either.
